docs/artifacts/three-leaks-one-hook.html

```
@@ -0,0 +1,136 @@
+<title>Three Leaks, One Hook</title>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)